cpp/iedriver/BrowserFactory.cpp

```diff
@@ -335,6 +335,40 @@ void BrowserFactory::LaunchBrowserUsingCreateProcess(PROCESS_INFORMATION* proc_i
   delete[] command_line;
 }
 
+bool DirectoryExists(std::wstring& dirName) {
+  DWORD attribs = ::GetFileAttributesW(dirName.c_str());
+  if (attribs == INVALID_FILE_ATTRIBUTES) {
+    return false;
+  }
+  return (attribs & FILE_ATTRIBUTE_DIRECTORY);
+}
+
+bool CreateUniqueTempDir(std::wstring &temp_dir) {
+  char temp[128];
+  std::wstring wtemp;
+
+  // get temporary folder for the current user
+  GetTempPathA(128, temp);
```

Comment:
We compile with UNICODE always set on, so we should avoid specific ASCII vs. Wide calls where possible. We should use `::GetTempPath` here, and save ourselves the manual conversion to `wstring`.

Reply:
Replaced these unicode APIs with general version APIs. Since `GetTempPath` take a `wchar_t` array as input, we still need a conversion from `wchar_t` array to `wstring` object.